setup.py

```diff
@@ -89,6 +89,9 @@
     "fsspec",
     # To get datasets from the Datasets Hub on huggingface.co
     "huggingface_hub<0.1.0",
+    # pydantic allows us to express simply validation patterns for dataset metadata and serialize object schemas
+    # for tasks descriptions.
+    "pydantic",
```

Comment:
Serialization is also possible with dataclasses.
Validation is a good reason to have an extra dependency, but we usually don't like adding extra dependencies to avoid issues in the future (breaking changes, bugs, etc.).
Maybe we can have a simple function for validation, without requiring an extra dependency